### ISSUE TITLE
dvc 1.0 --show-vega

### ISSUE THREAD

**Issue by andronovhopf** [iterative/dvc#3951]
I'm using DVC `1.0.0a7+9a55b9`. When I run

`dvc plots diff --target loss.csv --show-vega`, the output is being printed with a strange statement in the first line:

```
PlotTemplates.__init__
{
    "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
    "data": {
        "values": [
    {
        "index": 0,
        "loss": "0.35797476798407574",
        "rev": "workspace"
    },
    {
        "index": 1,
        "loss": "0.3108355691836759",
        "rev": "workspace"
    },
    {
        "index": 2,
        "loss": "0.30909985774265974",
        "rev": "workspace"
    },

etc.
```
Since I used the `--show-vega` flag last week with success, I'm wondering if something has changed?  @dmpetrov

**Comment by efiop** [comment]
Confirmed. Looking into it...

**Comment by efiop** [comment]
@andronovhopf Fixed on master and 1.0.0a8 will be released in ~15 minutes on pypi. Thanks for the feedback! :pray:

**Comment by andronovhopf** [comment]
@efiop awesome! i think the time it took you to fix it was faster than the time it took me to write a bash workaround :D
